pom.xml

```diff
@@ -58,6 +58,7 @@ limitations under the License.
     <!-- core dependency versions -->
     <bigtable.version>1.21.0</bigtable.version>
     <dropwizard.metrics.version>3.2.6</dropwizard.metrics.version>
+    <grpc-conscrypt.version>2.5.1</grpc-conscrypt.version>
```

Comment:
probably want to add to renovate config?

Reply:
good call, done!